vm/compiler/src/ledger/map/memory_map.rs

```diff
@@ -264,7 +276,7 @@ mod tests {
         assert!(map.iter().next().is_none());
 
         // Finish the current atomic write batch.
-        map.finish_atomic();
+        map.finish_atomic().unwrap();
```

Comment:
Maybe worth commenting on the safety of the `unwrap` here?

Reply:
it's in a `test` :wink:

Reply:
oh, so it is 😄